### ISSUE TITLE
Problem when working with Vue custom block

### ISSUE THREAD

**Issue by caxerx** [vatson/rollup-plugin-vuetify#32]
Hello, I have a problem when use this plugin with i18n custom block.
Seems vuetify plugin try to parse i18n custom block like a script. However, it's a json format and it parse failed.

```
$ rollup --config rollup.config.js

index.js → dist/index.js...
[!] (plugin vuetify) Error: The structure of the exported component can not be recognized. Please open an issue with a usage example at https://github.com/vatson/rollup-plugin-vuetify/issues
src\MyButton.vue?rollup-plugin-vue=i18n.0.json
Error: The structure of the exported component can not be recognized. Please open an issue with a usage example at https://github.com/vatson/rollup-plugin-vuetify/issues
    at findExport (C:\Users\caxerx\Desktop\atg-vue-component-configured\packages\default-button\node_modules\rollup-plugin-vuetify\src\transform.js:107:9)
    at findUseTarget (C:\Users\caxerx\Desktop\atg-vue-component-configured\packages\default-button\node_modules\rollup-plugin-vuetify\src\transform.js:111:21)
    at module.exports (C:\Users\caxerx\Desktop\atg-vue-component-configured\packages\default-button\node_modules\rollup-plugin-vuetify\src\transform.js:152:33)
    at extractAndTransform (C:\Users\caxerx\Desktop\atg-vue-component-configured\packages\default-button\node_modules\rollup-plugin-vuetify\src\plugin.js:11:10)
    at Object.transform (C:\Users\caxerx\Desktop\atg-vue-component-configured\packages\default-button\node_modules\rollup-plugin-vuetify\src\plugin.js:32:16)
    at ModuleLoader.addModuleSource (C:\Users\caxerx\Desktop\atg-vue-component-configured\packages\default-button\node_modules\rollup\dist\shared\rollup.js:17777:30)
    at ModuleLoader.fetchModule (C:\Users\caxerx\Desktop\atg-vue-component-configured\packages\default-button\node_modules\rollup\dist\shared\rollup.js:17831:9)
    at async Promise.all (index 3)
    at ModuleLoader.fetchStaticDependencies (C:\Users\caxerx\Desktop\atg-vue-component-configured\packages\default-button\node_modules\rollup\dist\shared\rollup.js:17855:34)
    at async Promise.all (index 0)
    at ModuleLoader.fetchModule (C:\Users\caxerx\Desktop\atg-vue-component-configured\packages\default-button\node_modules\rollup\dist\shared\rollup.js:17832:9)
    at async Promise.all (index 0)
    at ModuleLoader.fetchStaticDependencies (C:\Users\caxerx\Desktop\atg-vue-component-configured\packages\default-button\node_modules\rollup\dist\shared\rollup.js:17855:34)
    at async Promise.all (index 0)
    at ModuleLoader.fetchModule (C:\Users\caxerx\Desktop\atg-vue-component-configured\packages\default-button\node_modules\rollup\dist\shared\rollup.js:17832:9)
    at async Promise.all (index 0)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by vatson** [comment]
Hello @caxerx,

Could you provide an example of such component with custom block?

**Comment by caxerx** [comment]
The custom block is like this
```
<i18n lang="json">
{
  "en": {
    "hello": "hello world!"
  },
  "ja": {
    "hello": "こんにちは、世界！"
  }
}
</i18n>
```

I also configure the `rollup-plugin-vue` like this and use with [@intlify/rollup-plugin-vue-i18n](https://github.com/intlify/rollup-plugin-vue-i18n)
```
vue({
  customBlocks: ['i18n']
})
```

**Comment by caxerx** [comment]
There are similar problem when I use [extract css config](https://rollup-plugin-vue.vuejs.org/examples.html#extract-css).

When I configure `rollup-plugin-vue` like this:
```
vue({ css: false })
```

`rollup-plugin-vuetify` will try to parse the css block and cause an error.

**Comment by vatson** [comment]
@caxerx I found the cause of the error. It will be fixed in next patch release (0.2.3). Thanks for your contribution.
